configs/sst_display_hardware_multimedia-unwanted.yaml

```diff
@@ -11,6 +11,23 @@ data:
     - re2c
     # Totem is removed from RHEL 10, avoid usage in other components
     - totem-pl-parser
+    # Replaced by tools in libinput
+    - evemu
+    # BR for rubygem-clutter-gstreamer. Said rubygem shouldn't be in EL9. Also, superseeded by clutter-gst3.
+    - clutter-gst2
+    # Virtually unused even in Fedora, might have been used by spice once upon a time. Also, superseeded by opus.
+    - celt051
+    # Only used for some old games and emulators
+    - SFML
+    # Only used by some multimedia apps we don't ship
+    - libmpcdec
+    # Seems unused
+    - libsamplerate
+    # Only used by openal-soft examples, brings in other unwanted deps
+    - SDL_sound
+    # Removed in favor of pipewire(-pulse). Note that the SRPM with the same
+    # name should still be built for some packages, like pulseaudio-libs
+    - pulseaudio
```

Comment:
FYI see https://issues.redhat.com/browse/RHELBU-537 and https://src.fedoraproject.org/rpms/pulseaudio/pull-request/12

Reply:
Yep, I know. It was already part of sst_gpu-unwanted.yml, so I guess this could be in a follow-up MR once those are resolved?

Reply:
This should definitely remain here unwanted, but even better if we don't build it all.